azure-pipelines.yml

```diff
@@ -409,4 +408,59 @@ jobs:
           python ./tool_distributed_mutex.py unlock "$(MUTEX_URL)"
         name: 'Release_the_distributed_mutex'
         # this runs even if the job was canceled (only if the mutex was acquired by this job)
-        condition: and(always(), eq(variables['isMutexSet'], 'true'))
\ No newline at end of file
+        condition: and(always(), eq(variables['isMutexSet'], 'true'))
+
+  - job: Test_Scenarios
+    timeoutInMinutes: 360
+    strategy:
+      matrix:
+        Linux:
+          imageName: 'ubuntu-latest'
+          type: 'linux'
+          build_name: 'azcopy_linux_amd64'
+          container_name: 'testcontainer1'
+        Windows:
+          imageName: 'windows-latest'
+          type: 'windows'
+          build_name: 'azcopy_windows_amd64.exe'
+          container_name: 'testcontainer2'
+        MacOS:
+          imageName: 'macos-latest'
+          type: 'macos'
+          build_name: 'azcopy_darwin_amd64'
```

Comment:
build_name is not used in this pipeline. template files refines it based on the underlying environment,

Reply:
will fix it